components/camel-jetty/src/test/java/org/apache/camel/component/jetty/JettyMuteExceptionTest.java

```diff
@@ -41,13 +41,29 @@ public void testMuteException() throws Exception {
         }
     }
 
+    @Test
+    public void testDefaultMuteException() throws Exception {
+        HttpGet get = new HttpGet("http://localhost:" + getPort() + "/fooDefault");
+        get.addHeader("Accept", "application/text");
+        try (CloseableHttpClient client = HttpClients.createDefault();
+             CloseableHttpResponse response = client.execute(get)) {
+
+            String responseString = EntityUtils.toString(response.getEntity(), "UTF-8");
```

Comment:
Use `StandardCharsets.UTF_8` 

Obs.: I know the other test in there uses "UTF-8" (this is an old practice - we still have a few places doing this).

Reply:
Done.